source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -299,8 +300,14 @@ export default class WalletSendForm extends Component<Props, State> {
     return receiverField.value.length > 0;
   };
 
-  hasAssetValue = (asset: Field) => {
-    return get(asset, 'value', false);
+  isAddressFromSameWallet = () => {
+    const { isAddressFromSameWallet } = this.props;
+    const receiverField = this.form.$('receiver');
+    return (
+      this.hasReceiverValue() &&
```

Comment:
@aleksandardjordjeviciohk this part is repeated in two places - can we extract it into a method and reuse it in both?

Reply:
Fixed